api/v1alpha1/instaslice_types.go

```diff
@@ -115,6 +115,10 @@ type DiscoveredNodeResources struct {
 	// nodeResources represents the resource list of the node at boot time
 	// +required
 	NodeResources corev1.ResourceList `json:"nodeResources"`
+
+	// bootId represents the current boot id of the node
+	// +kubebuilder:validation:Required
+	BootID string `json:"bootId"`
```

Comment:
@rphillips is out today, but I got a ping from him about moving this to instaslice status. Would you like to consider that?